src/components/berryDexModal.html

```diff
@@ -174,31 +174,13 @@ <h5 class="modal-title text-light" data-bind="text: BerryType[$data]">Berry Stat
                             <th class="bg-dark">Wandering Pokémon</th>
                           </tr>
                           <tr>
-                            <th data-bind="text: 'Attracted by all Berries'">Base Wanderers</th>
-                            <td style="position: absolute; right: 0;"
-                            data-bind="visible: App.game.keyItems.hasKeyItem(KeyItemType.Pokerus_virus),
-                            tooltip: {
-                              title: 'These Pokémon do not gain EVs when attracted to the Farm.',
-                              trigger: 'hover',
-                              placement:'top',
-                              html: true,
-                            }">ⓘ</td>
+                            <th colspan="2" data-bind="text: 'Attracted by all Berries'">Base Wanderers</th>
                           </tr>
                           <tr id="wanderRow">
                             <td id="wanderPokemonList">
                               <ul class="list-inline">
-                              <!-- ko foreach: Berry.baseWander -->
-                              <li class="list-inline-item no-pokerus">
```

Comment:
`no-pokerus`  isn't being used anymore, so it can be removed from farm.less, unless you want to go the extra mile and disable it after getting the virus key item 🤔